tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1651,8 +1651,9 @@ def test_large_generation_multilingual(self):
         model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-large")
         model.to(torch_device)
 
-        ds = load_dataset("common_voice", "ja", split="test", streaming=True)
+        ds = load_dataset("mozilla-foundation/common_voice_6_1", "ja", split="test", streaming=True, token=True)
```

Comment:
does this dataset reauire a token? We don't have token (secret) set on the github actions for the CI (I believe).

Reply:
I confirm that we don't have token available on CI. If we really want to use this dataset, we will have to talk to the infra team regarding the security of having Hub token on github actions.

Reply:
Good catch! I'll move this change to a new PR, and then we can discuss with the infra team - we do need this dataset, the deprecated one (`common_voice`) doesn't work anymore.

Reply:
yes, move it to another PR is fine.


> the deprecated one (common_voice) doesn't work anymore.

I know, and once I moved to use  `mozilla-foundation/common_voice_11_0` and it doesn't require token - for which I don't know why and how long it will be the case

Reply:
see internal discussion https://huggingface.slack.com/archives/C034N0A7H09/p1698671476762139?thread_ts=1698669419.310549&cid=C034N0A7H09